src/reporter.rs

```diff
@@ -104,8 +104,10 @@ async fn spawn_terminal_reporter(
                 format!(
                     // example output
                     // 200       3280 https://localhost.com/FAQ
-                    "{} {:>10} {}\n",
+                    "{} {:>8} {:>8} {:>8} {}\n",
```

Comment:
Current output:
`200       83      333     3526 https://....`

Proposed:
`200       83l      333w     3526c https://....`

I'm open to other thoughts here, tho the short options for lines, words, and bytes according to the wc command seem pretty intuitive

Reply:
also, the heuristics output should be update to match